### PR TITLE
Save button is disabled on invalid names

### PR DESCRIPTION
**Pull request by Compro-Prasad**

### CODE REVIEW

File: src/resource-group-controller_impl.h [hunk 1]
```diff
@@ -15,41 +15,55 @@
...
 #include "resource-group.h"
+#include "resource-manager.h"
 
 #include "log.h"
 
 NAMESPACE__THITTAM__START
 
-class ResourceGroupControllerImpl :
-  public ResourceGroupController,
+class ResourceGroupControllerImpl : public ResourceGroupController,
```
Comment (joesteeve) [inline src/resource-group-controller_impl.h:24]: If you cannot have all superclasses in the same line, then indent all of them.

File: src/resource-group-controller_impl.h [hunk 2]
```diff
@@ -15,41 +15,55 @@
...
 
   /*--- ResourceGroupViewCallbacks interface ---*/
-  void view_save_pressed (void);
-  void view_cancel_pressed (void);
+  void
+  view_group_name_changed(void);
+  void
+  view_save_pressed(void);
+  void
+  view_cancel_pressed(void);
```
Comment (joesteeve) [inline src/resource-group-controller_impl.h:60]: This should be only for definition in .cxx files. In declarations, the return and function should be on the same line. Is the astyle doing this?

File: src/resource-group-controller_impl.h (hunk 1 above)
Comment (Compro-Prasad) [reply src/resource-group-controller_impl.h]: That is indented.

File: src/resource-group-controller_impl.h (hunk 2 above)
Comment (Compro-Prasad) [reply src/resource-group-controller_impl.h:60]: Yes, probably.

**Pull request merged**